.circleci/config.yml

```diff
@@ -0,0 +1,50 @@
+orbs:
+  slack: circleci/slack@3.4.1
+version: 2.1
+jobs:
+  test:
+    machine:
+      image: ubuntu-1604:201903-01
+    steps:
+      - checkout
+      - run: docker run --name postgres -p 5432:5432 -d postgres:10.6
+      - run: wget https://packages.erlang-solutions.com/erlang-solutions_2.0_all.deb && sudo dpkg -i erlang-solutions_2.0_all.deb
+      - run: sudo apt-get update
+      - run: sudo apt-get install elixir esl-erlang
+      - run: MIX_ENV=test mix local.hex --force
+      - run: MIX_ENV=test mix local.rebar --force
+      - run: MIX_ENV=test mix deps.get
+      - run: MIX_ENV=test mix coveralls
+  build:
+    machine:
+      image: ubuntu-1604:201903-01
+    steps:
+      - checkout
+      - run: docker-compose -f docker/docker-compose-secrets.yml -f docker/docker-compose.yml -f docker/docker-compose.dev.yml build --no-cache oms-core-elixir
+  docker-build-and-push:
+    machine:
+      image: ubuntu-1604:201903-01
+    steps:
+      - checkout
+      - run: docker build  --tag aegee/oms-core-elixir:$(node -p "require('./package.json').version") --tag aegee/oms-core-elixir:latest -f docker/oms-core-elixir/Dockerfile .
+      - run: docker login --username $DOCKER_LOGIN --password $DOCKER_PASSWORD
+      - run: docker push aegee/oms-core-elixir:$(node -p "require('./package.json').version")
```

Comment:
isn't there a pipeline env variable that lets you get this metadata? see here as [example of gitlab](https://docs.gitlab.com/ee/ci/variables/predefined_variables.html)

Reply:
I was thinking to set the env variable initially, like `export VERSION=$(node -p "require('./package.json')`, but that's tricky and I don't remember why I did this instead. anyway it's the same way in other services.

and afaik, there's no such variable in build-in variables unfortunately

Reply:
https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables